lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -0,0 +1,121 @@
+import { createElement } from 'preact';
+import propTypes from 'prop-types';
+import { useContext, useEffect, useState, useRef } from 'preact/hooks';
+
+import { Config } from '../config';
+import { submitGrade } from '../utils/grader-service';
+import { formatToNumber, validateGrade } from '../utils/validation';
+import ErrorDialog from './ErrorDialog';
+import Spinner from './Spinner';
+import ValidationMessage from './ValidationMessage';
+
+/**
+ * A form with a single input field and submit button for an instructor to
+ * save a students grade.
+ */
+
+export default function SubmitGradeForm({ disabled, student }) {
+  // for validation
```

Comment:
Given that the next two lines have inline comments, I think this comment can probably go.